Pointwise.lua

```diff
@@ -69,5 +88,7 @@ end
 
 function Pointwise:clearState()
    self:clearDesc()
+   self._gradOutput = nil
+   nn.utils.clear(self, 'activDesc')
```

Comment:
yes it's better to use clear() to allow buffer sharing, see https://github.com/torch/nn/pull/526